lib/DB.js

```diff
@@ -5,11 +5,11 @@ class DB {
         this.sequelize = sequelize;
         this.models = models;
     }
-
+    
```

Comment:
This should not happen because `prettier` should remove useless space characters. Even if you don't have it setup in your IDE, the `precommit` hook should not let that pass through. It results in having a diff on file for no reason. Is your local project dependencies up-to-date? Did you see `precommit` hook running when you submit your PR? Asking because there might be an issue with that.

Reply:
My dependencies are up-to-date, I didn't noticed if the `precommit` hook run ok, sorry.